### PR TITLE
Feature envelope improve

### PR DESCRIPTION
**Pull request by cameronc137**
I've made some changes to the analysis codes, and made a new version of reader that is done manually.

### CODE REVIEW

**Comment by cameronc137** [comment]
This branch will contain the updated collimator 2 and 4 and 5 radial cut criteria (in pruneTreeEnvelope, the GDML stuff will go straight to a develop fork branch), as well as the lintels, once that work is done. This will be used to track the developments of the reader and pruneTreeEnvelope software as it is optimized for envelope production. It should have been merged into develop a while ago, but this will work for now.

**Comment by wdconinc** [comment]
What is the status of this branch and pull request?

**Comment by cameronc137** [comment]
It's ready to merge

File: macros/runexample_epelastic_envelope.mac [hunk 1]
```diff
@@ -3,7 +3,7 @@
 # Macrofile
 # This must be called before initialize
 /remoll/setgeofile geometry/mollerMother_acceptanceDefinition_includeCol4.gdml
-/remoll/parallel/setfile geometry/mollerParallel.gdml
+/remoll/parallel/setfile geometry/mollerParallel_envelopes.gdml
```
Comment (cameronc137) [inline macros/runexample_epelastic_envelope.mac]: This parallel geometry should be moved to the parallel folder and have the duplicate detectors removed. It should also have the z positions updated with the new origin information.

**Comment by cameronc137** [comment]
Should the geometry used by this envelope business be brought up to date with the new-experiment geometry z locations and parallel world strategy before the merge, or left as a task for after?

**Comment by wdconinc** [comment]
Develop is now using Z centered at hall center, so for consistency that would be great.

**Comment by wdconinc** [comment]
@cameronc137 Not merging yet until this has been updated to include shifted z positions with z = 0 at the pivot.

**Comment by cameronc137** [comment]
Everything except for the final commit (september, includes geometry files) can be merged in with no conflicts (except the mollerParallel_envelopes.gdml will have to be deleted, its obsolete already).

Geometry updates are to be compared and merged in from the new branch and PR: #286

**Comment by wdconinc** [comment]
Please delete mollerParallel_envelopes.gdml and then I'll merge.

**Comment by cameronc137** [comment]
Did this whole branch get merged in? From the time line here it looks like only one commit got merged in?

**Comment by wdconinc** [comment]
It looks merged. No mollerParallel_envelope in develop.

**Pull request merged**